src/router/mod.rs

```diff
@@ -106,7 +106,7 @@ impl Router {
     }
 
     fn handle_routing_update(&mut self, logger: &Logger, routing_response: &RoutingResponse) {
-        if routing_response.height <= self.routing_height {
+        if routing_response.height < self.routing_height {
```

Comment:
Because the client would already have processed routing info at that height, so something fishy is going on? Are you seeing this happen?

Reply:
It was reported [here](https://github.com/helium/gateway-rs/issues/30#issuecomment-816880282) in an unrelated issue.

Reply:
That's an actual problem with the gateway service then.. and the warning is correct. It's not terminal but it does indicate the gateway service is doing something not quite right

Reply:
Appropriate warning then!